src/org/jitsi/service/neomedia/RawPacket.java

```diff
@@ -1888,6 +1893,24 @@ public boolean setVersion()
         return true;
     }
 
+    /**
+     * Whether to skip packet statistics.
+     * @return returns true if we want to skip stats for this packet.
+     */
+    public boolean isSkipStats()
+    {
+        return skipStats;
+    }
+
+    /**
+     * Changes the skipStats flag.
+     * @param skipStats the new value.
+     */
+    public void setSkipStats(boolean skipStats)
```

Comment:
I don't see this used anywhere. Are we calling it from Jigasi?

Reply:
Yes.